### PR TITLE
get initializers working for mods

### PR DESCRIPTION
**Pull request by xithan**
We used to have our initializers in `mod/mymod/initializers`. At the end of February (the last time we saw a airbrake message) the expected initializer directory was moved to `mod/config/initializers`. I fixed it to `mod/*/config/initializers`. We still have to move our initializers to the config directory

### CODE REVIEW

File: card/config/initializers/01_core_extensions/object.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+module CoreExtensions
+  module Object
+    def deep_clone
```
Comment (ethn) [inline card/config/initializers/01_core_extensions/object.rb]: I can't seem to count 18 lines here, @houndci-bot

File: card/mod/01_core/chunk/include.rb [hunk 2]
```diff
@@ -68,7 +68,7 @@ def interpret_piped_options list_string, items
     end
 
     def option_string_to_hash list_string, options_hash, style_hash
-      Hash.new_from_semicolon_attr_list(list_string).each do |key, value|
+      each_option(list_string) do |key, value|
```
Comment (ethn) [inline card/mod/01_core/chunk/include.rb]: YES!

**Comment by ethn** [comment]
most of these hound comments are really easy. could be worth a quick fix.

Regardless, merge at will :)

**Comment by xithan** [comment]
all the missing final newlines are fixed. Looks like the hound or github doesn't notice whitespace changes

**Pull request merged**